src/qibocal/protocols/characterization/RB/std_rb_unrolling.py

```diff
@@ -0,0 +1,200 @@
+            ExecutionParameters(
+                nshots=params.nshots,
+                relaxation_time=params.relaxation_time,
+                acquisition_type=AcquisitionType.DISCRIMINATION,
+                averaging_mode=AveragingMode.CYCLIC,
+            ),
+        )
+
+        counts_depths = {}
+        for depth in depths:
+            starts = [pulse.start for pulse in ro_pulses[depth]]
+            counts = {}
+            for pulse in ro_pulses[depth]:
+                counts[pulse] = starts.count(pulse.start)
+            counts_depths[depth] = counts
+
+        # average msr, phase, i and q over the number of shots defined in the runcard
+        j = 0
+        for depth in depths:
+            for ro_pulse in counts_depths[depth].keys():
+                for i in range(counts_depths[depth][ro_pulse]):
+                    probs = results[ro_pulse.serial][i].probability(0)
+                    qubit = ro_pulse.qubit
+                    data.register_qubit(qubit, 0, len(circuits[j]), probs)
+                    j += 1
+
+    return data
+
+
+def _fit(data: StdRBData) -> StdRBResults:
+    """Post-processing for Standard RB."""
+
+    qubits = data.qubits
+
+    fitted_parameters = {}
+    fidelities_primitive = {}
+    fidelities = {}
+    average_errors_gate = {}
+
+    for qubit in qubits:
+        qubit_data = data[qubit]
+
+        x = qubit_data.length
+        y = qubit_data.probabilities
+
+        a_guess = np.max(y) - np.mean(y)
+        p_guess = 0.9
+        b_guess = np.mean(y)
+
+        pguess = [a_guess, p_guess, b_guess]
+        try:
+            popt, pcov = curve_fit(
+                utils.RB_fit,
+                x,
+                y,
+                p0=pguess,
+                maxfev=100000,
+                # bounds=((-np.inf, 0, -np.inf), (np.inf, 1, np.inf)),
+                bounds=((0, 0, 0), (1, 1, 1)),
+            )
+
+            p = popt[1]
+            fidelity_primitive = 1 - ((1 - p) / 2)
+            # Divide infidelity by magic number
+            magic_number = 1.875
+            infidelity = (1 - p) / magic_number
+            fidelity = 1 - infidelity
+            average_error_gate = infidelity * 100
+
+            fitted_parameters[qubit] = popt
+            fidelities[qubit] = fidelity
+            fidelities_primitive[qubit] = fidelity_primitive
+            average_errors_gate[qubit] = average_error_gate
+
+        except:
+            log.warning("RB_fit: the fitting was not succesful")
+            fidelities[qubit] = 0.0
+            fidelities_primitive[qubit] = 0.0
+            average_errors_gate[qubit] = 0.0
+            fitted_parameters[qubit] = [0] * 3
+
+    return StdRBResults(
+        fidelities, fidelities_primitive, average_errors_gate, fitted_parameters
+    )
+
+
+def _plot(data: StdRBData, fit: StdRBResults, qubit):
+    """Plotting function for Standard RB."""
+    return utils.plot(data, fit, qubit)
+
+
+StdRB_unrolling = Routine(_acquisition, _fit, _plot)
```

Comment:
I'd keep following naming conventions
```suggestion
std_rb_unrolling = Routine(_acquisition, _fit, _plot)
```